### PR TITLE
feat(rancher): add dashboard page and pooling on rancher

### PR DESCRIPTION
**Pull request by lionel95200x**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-12775-rancher-epic` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-12785
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

- Add dashboard page for Rancher Project
- Setup Pooling of rancher to display message statut

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
Complete the PR checklist and description

File: packages/manager/apps/pci-rancher/src/components/TileBlock/TileBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { OsdsText, OsdsDivider } from '@ovhcloud/ods-components/react';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+
+export type TileBlockProps = {
+  label: string;
+};
```
Comment (ghyenne) [inline packages/manager/apps/pci-rancher/src/components/TileBlock/TileBlock.tsx:8]: I think that the typing of children is missing

File: packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -37,21 +60,33 @@ const Dashboard: React.FC<DashboardLayoutProps> = ({ tabs }) => {
 
   return (
     <>
-      <div className="py-4">
+      <div className="py-4 ">
```
Comment (ghyenne) [inline packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/Dashboard.tsx:63]:
```suggestion
      <div className="py-4">
```

File: packages/manager/apps/container/src/container/legacy/server-sidebar/universe/public-cloud/pci-menu.ts [hunk 3]
```diff
@@ -34,6 +34,7 @@ export const features = [
   'data-processing',
   'logs-data-platform',
   'horizon',
+  'rancher',
```
Comment (anooparveti) [inline packages/manager/apps/container/src/container/legacy/server-sidebar/universe/public-cloud/pci-menu.ts:37]:
```suggestion
  'pci-rancher',
```

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/services/publicCloud.ts [hunk 4]
```diff
@@ -196,6 +196,7 @@ const pciNode: Node = {
       id: 'pci-containers',
       translation: 'sidebar_pci_containers',
       features: [
+        'rancher',
```
Comment (anooparveti) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/services/publicCloud.ts:199]:
```suggestion
        'pci-rancher',
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/pci-rancher/src/pages/dashboard/Tabs1.tsx [hunk 5]
```diff
@@ -2,7 +2,9 @@ import React from 'react';
 
 function Tabs1() {
   return (
-    <div className="grid xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 py-6"></div>
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/pci-rancher/src/pages/dashboard/Tabs1.tsx:5]: Do you use this Tabs1 ? you have Dashboard.tsx no ?

File: packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail.tsx [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+            <OsdsText
+              size={ODS_TEXT_SIZE._400}
+              level={ODS_TEXT_LEVEL.heading}
+              color={ODS_THEME_COLOR_INTENT.text}
+            >
+              {t('consumption')}
+            </OsdsText>
+            <OsdsDivider separator />
+            <TileBlock label={t('service_level')}>
+              {rancher.currentState.plan}
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail.tsx:118]: no need to use OsdsText ?

File: packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail.tsx [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+            </TileBlock>
+            {vCpus && (
+              <TileBlock label={t('count_cpu_orchestrated')}>
+                <div>{rancher.currentState.usage?.orchestratedVcpus}</div>
+                {dateUsage && (
+                  <div className="mt-3">
+                    {t('last_update_date', {
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/RancherDetail.tsx:125]: no need to use OsdsText ?

**Review by anooparveti — CHANGES_REQUESTED** [review]
Please complete the PR description with appropriate info.
* No corresponding ticket number is mentioned
* Target branch name is wrong

**Comment by aboungnaseng-ovhcloud** [comment]
- Missing keys in rancherTaskMessage (we have console warning)
- In the delete modal you can click on delete button while the button is disabled
- Wrong police under  under Rancher version
-  Wrong police under  under "Number of CPUs" too
- "Ip authorized" in the tab is clickable
- In the "Ip authorized" tab, the chip "coming soon" is also missing
- The font is not good in the description of “general informations”

**Pull request merged**